test/jdk/ProblemList.txt

```diff
@@ -587,7 +587,6 @@ sun/security/pkcs11/sslecc/ClientJSSEServerJSSE.java            8161536 generic-
 sun/security/tools/keytool/ListKeychainStore.sh                 8156889 macosx-all
 
 javax/net/ssl/SSLEngine/TestAllSuites.java                      8298874 generic-all
-javax/net/ssl/SSLEngine/IllegalRecordVersion.java               8298873 generic-all
```

Comment:
I have closed 8298873 as duplicate of this bug. Can you please update IllegalRecordVersion test to list 8299870 under `@bug`.

Reply:
Since 8299870 is test only bug now, we don't need to list it.